shaptools/hana.py

```diff
@@ -591,10 +613,11 @@ def set_ini_parameter(
         user_name = kwargs.get('user_name', None)
         user_password = kwargs.get('user_password', None)
 
-        self._manage_ini_file(parameter_str=parameter_str, database=database,
-                              file_name=file_name, layer=layer, layer_name=layer_name,
-                              set_value=True, reconfig=reconfig, key_name=key_name,
-                              user_name=user_name, user_password=user_password)
+        self._manage_ini_file(
```

Comment:
why are you adding a newline here? I mean seems unrelated to PR IMHO :grin:

Reply:
Yes... Just it looks like much better this way (following python patterns). Otherwise there are lines with spaces instead of tabs...